Forge server throws error at launch.

What I did:
* Build the image from the forge Dockerfile + scripts.
* Run it as `docker run -it -e VERSION="1.10.2-recommended" -p 25565:25565 -v $HOME/modserver:/data mcmods`

Got this:

```
Downloading promotions manifest...
Downloading Forge server (1.10.2-12.18.3.2185)...
Installing Forge server...
Clean up...
A problem occurred running the Server launcher.java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at net.minecraftforge.fml.relauncher.ServerLaunchWrapper.run(ServerLaunchWrapper.java:62)
	at net.minecraftforge.fml.relauncher.ServerLaunchWrapper.main(ServerLaunchWrapper.java:31)
Caused by: java.lang.ClassCastException: class jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to class java.net.URLClassLoader (jdk.internal.loader.ClassLoaders$AppClassLoader and java.net.URLClassLoader are in module java.base of loader 'bootstrap')
	at net.minecraft.launchwrapper.Launch.<init>(Launch.java:34)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	... 6 more```

--- a/forge/Dockerfile
+++ b/forge/Dockerfile
@@ -4,7 +4,7 @@
 #
 
 # Pull base image
-FROM openjdk:jre-slim
+FROM openjdk:8-jre-slim
 
 # Install dependencies
 RUN apt-get update; \

